### PR TITLE
Resolves #1681

### LINKED ISSUE #1681: Fill in "-PolicyFilePath" description
The description for the "-PolicyFilePath" parameter says "{{Fill PolicyFilePath Description}}". This should be filled in.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9ab88b71-84f4-7777-7791-06869c5d7caf
* Version Independent ID: f56d8bcc-0c53-0338-3e58-f30c45071651
* Content: [Set-ProcessMitigation (processmitigations)](https://docs.microsoft.com/en-us/powershell/module/processmitigations/Set-ProcessMitigation?view=win10-ps#feedback)
* Content Source: [docset/windows/processmitigations/Set-ProcessMitigation.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/windows/processmitigations/Set-ProcessMitigation.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Add description for *PolicyFilePath parameter.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1681

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Gets the current process mitigation for "notepad.exe" from the registry and then
 
 ### Example 2
 ```
-PS C:\> set-ProcessMitigation -file settings.xml
+PS C:\> set-ProcessMitigation -File settings.xml
```
Comment (get-itips) [inline docset/windows/processmitigations/Set-ProcessMitigation.md]:
```suggestion
PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
```

Comment (tseward) [reply docset/windows/processmitigations/Set-ProcessMitigation.md]: Agree, there is no `-File` parameter.

**Comment by velkovb** [comment]
Good catch. Did not pay attention to it.

**Pull request merged**